### ISSUE TITLE
Can't control my MYQ.

### ISSUE THREAD

**Issue by PhiltorX** [DanCech/myq-bridge#1]
I'm wanting to use this with my Chamberlain MyQ MYQ-G0201 and UDI ISY994i. When I use the /doors URL it returns with only []. if I use /doors/Garage+Door/staus it says {"code": 3, "error": "Door Garage Door not found"}. Any advice?

**Comment by DanCech** [comment]
Can you paste the log output from the server process?

**Comment by PhiltorX** [comment]
I don't know where the log is kept. FWIW, I got similar results when I tried https://github.com/Einstein42/myq-garage/blob/master/myq-garage.py.

**Comment by DanCech** [comment]
When you run myq.py you should get output like:

```
$ ./myq.py
2016-02-24 18:31:06,243 Starting
2016-02-24 18:31:06,245 Getting Doors
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
```

then when you hit `http://127.0.0.1:5000/doors/` something like:

```
2016-02-24 18:31:52,917 Main Door is Closed. Last changed at Wed 24 Feb 2016 16:53:13
127.0.0.1 - - [24/Feb/2016 18:31:52] "GET /doors/ HTTP/1.1" 200 -
```

(In my case I have a single door called "Main Door", if you have multiple doors there should be a status line logged for each door.)

The fact that you're getting `[]` back from `/doors/` indicates that the API output doesn't contain anything the script can recognize as a door.

I'll see if I can add a debug flag so it can print out the responses from the API, that might help track things down.

**Comment by PhiltorX** [comment]
My garage door is simply named "Garage Door". When I do /doors/ I get: 

./myq.py
2016-02-24 18:40:42,528 Starting
2016-02-24 18:40:42,547 Getting Doors
- Running on http://192.168.1.8:5000/ (Press CTRL+C to quit)
  192.168.1.108 - - [24/Feb/2016 18:41:22] "GET /doors/ HTTP/1.1" 200 -
  192.168.1.108 - - [24/Feb/2016 18:41:22] "GET /favicon.ico HTTP/1.1" 404 -

**Comment by DanCech** [comment]
Yeah, that jives with it not seeing any doors in the API output

**Comment by PhiltorX** [comment]
I just finished redoing python2.7, flask, and request and still get the same results. Also tried it on another RPI2.

**Comment by DanCech** [comment]
Ok, I just pushed a commit that adds support for a debug flag in the MyQ section of the config, please update your myq.cfg so it looks like:

```
[MyQ]
username = user@domain.com
password = password
debug = true

[Flask]
...
```

Then run myq.py again and email me the output (the password and security tokens will be masked but it still contains things like unit serial numbers etc).

**Comment by DanCech** [comment]
Fixed by d50bca8decf76ab403a7abfa0af837da1005484f
